### DISCUSSION TITLE
Tinyurl.com/goofguardian is not working

### DISCUSSION THREAD

**Discussion by Wynfhi** [3kh0/ext-remover#388] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
when I go to tinyurl.com/goofguardian I get this
![Screenshot 2023-01-10 9 44 42 AM](https://user-images.githubusercontent.com/68559143/211596641-ba778c7b-3b79-4478-b08d-9ad4c66d4ef6.png)

**Comment by Wynfhi** [comment, marked as answer]
Going through the gogardian files to get a updated version for viewing I came across a belief that they may have decided to block the link to viewing the file itself.
Heres the results 
```
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>Z2BQQ0DYSQZAAGV7</RequestId>
<HostId>LsO+OTd36a1B13rQLHQE+UzDl8NYp2nBcWLUMTRcBh0bSwKCs5p64gzvPge4phcjjtMzdQiqN4QOzCexfPrb4w==</HostId>
</Error>
```
and heres the link https://crx.goguardian.com/extension-m-3.0.6455.1.crx-stable.crx

**Reply by Aka-but-nice** [reply]
Yes I know that they blocked the XSS file, I'm smarter than you think, and I already talked about this, it can't be fixed until we find more unsafe uses of innerHTML in GoGuardian,  So this won't work.

**Reply by Wynfhi** [reply]
rip

**Reply by Aka-but-nice** [reply]
and also, <Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>Z2BQQ0DYSQZAAGV7</RequestId>
<HostId>LsO+OTd36a1B13rQLHQE+UzDl8NYp2nBcWLUMTRcBh0bSwKCs5p64gzvPge4phcjjtMzdQiqN4QOzCexfPrb4w==</HostId>
</Error> is not what's causing it, they removed that file, that code is the code from the block page

**Reply by Aka-but-nice** [reply]
all the tinyurl.com/ extension links are block pages, with XSS attacks

**Reply by Aka-but-nice** [reply]
> Going through the gogardian files to get a updated version for viewing I came across a belief that they may have decided to block the link to viewing the file itself. Heres the results
> 
> ```
> <Error>
> <Code>AccessDenied</Code>
> <Message>Access Denied</Message>
> <RequestId>Z2BQQ0DYSQZAAGV7</RequestId>
> <HostId>LsO+OTd36a1B13rQLHQE+UzDl8NYp2nBcWLUMTRcBh0bSwKCs5p64gzvPge4phcjjtMzdQiqN4QOzCexfPrb4w==</HostId>
> </Error>
> ```
> 
> and heres the link https://crx.goguardian.com/extension-m-3.0.6455.1.crx-stable.crx

Also, they didn't block the link, they DELETED it, that was the only trace of XSS in GoGuardian

**Reply by Arthura0121** [reply]
so what are we supposed to do?

**Reply by Aka-but-nice** [reply]
Really, just use proxies, and wait.

**Reply by Aka-but-nice** [reply]
@Arthura0121 try killcurly it's listed at https://github.com/3kh0/ext-remover

**Reply by Arthura0121** [reply]
alr
